client/qiskit_serverless/core/clients/local_client.py

```diff
@@ -112,11 +111,16 @@ def run(
             **(saved_program.env_vars or {}),
             **{OT_PROGRAM_NAME: saved_program.title},
             **{"PATH": os.environ["PATH"]},
-            **{ENV_JOB_ARGUMENTS: json.dumps(arguments, cls=QiskitObjectsEncoder)},
         }
 
+        with open("arguments.serverless", "w", encoding="utf-8") as f:
```

Comment:
How would you see to move the name of the file to a constant? Probably with the migration to object storage we are going to need to modify it and having only one reference would be cool.